NeoFlux/appsettings.PrivateAllCode.json

```diff
@@ -13,6 +13,7 @@
       "http://ec2-54-89-238-202.compute-1.amazonaws.com:10332",
       "http://ec2-54-172-121-105.compute-1.amazonaws.com:10332",
       "http://ec2-35-170-246-74.compute-1.amazonaws.com:10332"
-    ]
+    ],
+    "PythonNode": "http://ec2-18-232-80-57.compute-1.amazonaws.com:8080/v1/"
```

Comment:
If this is now required for everything, consider adding it into the main config, so you know it has to be changed (the appsettins.json itself)